examples/plane/limiters_advection.jl

```diff
@@ -189,33 +189,33 @@ for (k, ne) in enumerate(ne_seq)
             Geometry.UVVector(uu, uv)
         end
 
-        # Compute min_Q[] and max_Q[] that will be needed later in the stage limiter
+        # Compute min_q[] and max_q[] that will be needed later in the stage limiter
         space = parameters.space
         n_elems = Topologies.nlocalelems(space)
         topology = space.topology
 
-        neigh_elems_Q_min = Array{Float64}(undef, 8)
-        neigh_elems_Q_max = Array{Float64}(undef, 8)
+        neigh_elems_q_min = Array{Float64}(undef, 8)
+        neigh_elems_q_max = Array{Float64}(undef, 8)
 
         for e in 1:n_elems
-            Q_e = Fields.slab(y.ρq, e)
+            q_e = Fields.slab(y.ρq ./ y.ρ, e)
 
-            Q_e_min = minimum(Q_e)
-            Q_e_max = maximum(Q_e)
+            q_e_min = minimum(q_e)
+            q_e_max = maximum(q_e)
             neigh_elems = Topologies.neighboring_elements(topology, e)
             for i in 1:length(neigh_elems)
                 if neigh_elems[i] == 0
-                    neigh_elems_Q_min[i] = +Inf
-                    neigh_elems_Q_max[i] = -Inf
+                    neigh_elems_q_min[i] = +Inf
+                    neigh_elems_q_max[i] = -Inf
                 else
-                    neigh_elems_Q_min[i] =
-                        Fields.minimum(Fields.slab(y.ρq, neigh_elems[i]))
-                    neigh_elems_Q_max[i] =
-                        Fields.maximum(Fields.slab(y.ρq, neigh_elems[i]))
+                    neigh_elems_q_min[i] =
+                        Fields.minimum(Fields.slab(y.ρq ./ y.ρ, neigh_elems[i]))
```

Comment:
You're doing `y.ρq ./ y.ρ` at every iteration of the loop

Reply:
What would you suggest we should do? We need this on each element slab

Reply:
One thing I can think of is returning `q` only instead of `ρq` from the limiters routine. But that will require some more changes